package.json

```diff
@@ -5,7 +5,15 @@
   "description": "A starter project for AngularJS",
   "repository": "https://github.com/angular/angular-seed",
   "license": "MIT",
+  "dependencies": {
+    "angular": "^1.5.9",
+    "angular-loader": "^1.5.9",
+    "angular-route": "^1.5.9",
+    "html5-boilerplate": "0.0.1",
+    "shelljs": "^0.7.5"
```

Comment:
We need grunt in the angular 1 project to generate the production distribution but that doesn't stop grunt from being a `devDependency`. Unless we ship `shelljs` as part of the application I don't think I would call is a `dependency`.